nixos/modules/services/web-apps/glitchtip.nix

```diff
@@ -255,6 +294,30 @@ in
         };
       };
 
+    systemd.sockets = lib.mkIf cfg.nginx.createLocally {
+      glitchtip.socketConfig = {
+        ListenStream = "/run/glitchtip/glitchtip.sock";
+        SocketUser = "nginx";
+      };
+    };
+
+    services.nginx = lib.mkIf cfg.nginx.createLocally {
+      enable = true;
+      upstreams.glitchtip.servers."unix:/run/glitchtip/glitchtip.sock" = { };
```

Comment:
is there a reason to create an `upstream` if it's used only once and no other options are set?